widgetastic_manageiq/__init__.py

```diff
@@ -3437,17 +3437,19 @@ def all_entity_names(self):
         # get_all uses self.entity_names, which handles versioned name query
         return [e.name for e in self.get_all(surf_pages=True)]
 
-    def get_all(self, surf_pages=False):
+    def get_all(self, surf_pages=False, slice=None):
```

Comment:
I'm not entirely sure this change is necessary at all.
You can have to same entities using either apply() method or get_all(surf_pages=False)[:2].

since this is useful pattern, I'd add 'first_n_entities(n) method to entity class. it can be similar to first_entity() method.

Reply:
I initially did something like get_all(surf_pages=False)[:2]. It was suggested that I put the slice inside of the function so we would not build each entity and then just return the ones we need. The way I have it here saves us from building all entities. It's the primary reason for this PR.

Reply:
Discussed offline, change to `get_all` was accepted. Most issue  by reviewers had to do with the naming and not the optimization provided.